### ISSUE TITLE
Crash on operations using dbus calls

### ISSUE THREAD

**Issue by andreldm** [andreldm/thunar#12]
Some operations crash on dbus calls:
```
(lt-thunar:5677): GLib-GIO-CRITICAL **: g_dbus_proxy_call_finish_internal: assertion 'error == NULL || *error == NULL' failed
[1]    5677 segmentation fault (core dumped)  thunar
```
Reproducible with:
* Rename files (right click rename & F2 hotkey)
* Copy and paste (right click context menu)
* Send to desktop
* Restore file or folder from trash

**Comment by alexxcons** [comment]
Reproducible as well with drag + drop

**Comment by andreldm** [comment]
All these operations involve copying files and they crash at the same point:
thunar-thumbnail-cache.c:169
thunar_thumbnail_cache_dbus_call_copy_finish

**Comment by andreldm** [comment]
Likewise move operations also crash on thunar_thumbnail_cache_dbus_call_move_finish.
There's a gdbus-ported tumbler available: https://git.xfce.org/users/ali/tumbler/
Let's see how well it plays with thunar-gtk3.

**Comment by andreldm** [comment]
No luck with Ali's tumbler port.
The problem seems to be  thunar_thumbnail_cache_dbus_call_[copy/move]_finish returning false, but when error->message is accessed or error is cleared it wreaks havoc.
Commenting the subsequent g_printerr and g_clear_error avoids the crash, of course that's not the right thing to do.

**Comment by frapa** [comment]
I had a look at this bug (thanks andreldm for the infos). It seems g_dbus_proxy_call_finish is returning uninitialized memory (the message field of the GError struct is not initialitzed). I have no idea how we could solve this bug, since g_dbus_proxy_call_finish is an function of a library we have no control over.

**Comment by andreldm** [comment]
Fixed in e2dfd9d by Skunnyk
